sale_variant_configurator/models/sale_order.py

```diff
@@ -64,6 +64,10 @@ class SaleOrderLine(models.Model):
     _name = "sale.order.line"
 
     product_id = fields.Many2one(required=False)
+    product_uom = fields.Many2one(
+        default=lambda self: self.env['product.uom'].search([],
```

Comment:
Why this strange indentation? When lambda is a bit large, better to extract it to a method.